support-frontend/app/services/AuthenticationService.scala

```diff
@@ -44,8 +45,10 @@ class AsyncAuthenticationService(
     authenticateUser(requestHeader)
       .map(user => Option(user))
       .handleError { err =>
-        // TODO: inspect errors that this is generating and see if we want log level and/or message to be dependent on error type.
-        logger.info("unable to authenticate user", err)
+        // User shouldn't necessarily be signed in,
+        // therefore, don't log failure to authenticate as a result of this with log level ERROR.
+        if (isUserNotSignedInError(err)) logger.info(s"unable to authorize user - $err")
+        else logger.error(s"unable to authorize user - $err")
```

Comment:
Is this error log going to contain any PII? afaik, support-frontend deliberately scrubs error logging because there's no need for sentry to store PII https://github.com/guardian/support-frontend/pull/494

Reply:
oh yes good point, should be `SafeLogger.error(scrub"blah $info etc")`

Reply:
ah thanks!